documents/src/pages/elements/tree-select.md

```diff
@@ -520,6 +520,24 @@ This control will save the current selection, and result of the firing of the `v
 
 This control will abort any current selection changes and go back to the tree state when first opened.
 
+## Manipulate item property
+
+Tree Select component supports functionality for updating and reading data to manipulate items properties. These are used in conjunction with [TreeManager](./custom-components/utils/tree-manager) from `treeManager` property and [TreeNode](./custom-components/utils/tree-node) which can be retrieved from `getTreeNode` through `treeManager` property.
+
+```javascript
+const treeSelect = document.querySelector('ef-tree-select');
```

Comment:
let's provide a more concrete example. Check Tree Node doc in `feat/tree-node/docs-tree-node` for a real example.

Reply:
Added.